docs/Makefile

```diff
@@ -86,9 +89,9 @@ linkcheck:
 	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
 	@echo
 	@echo "Link check complete; look for any errors in the above output " \
-	      "or in $(BUILDDIR)/linkcheck/output.txt."
+		  "or in $(BUILDDIR)/linkcheck/output.txt."
```

Comment:
What's the reason for these indentation changes? `make` requires tab indentation to indent a block but the spaces look very much intentional to align it nicely.